README.rst

```diff
@@ -73,6 +73,20 @@ Some Python HTTP clients that you can use to retrieve HTML include `requests`_,
 .. _urllib.request module: https://docs.python.org/3/library/urllib.request.html
 
 
+Test data
+---------
+
+As a precondition for inclusion in this library, each recipe scraper is required to include test coverage to demonstrate that it can accurately retrieve recipe information from the website it relates to.
+
+To achieve that, the development source repository includes a minimal number of test HTML files from the public web -- often only one or two per recipe website -- to confirm the scraper's behaviour.  These test files are not included in the [published releases](https://pypi.org/project/recipe-scrapers/) of the library.
+
+If you would prefer for any of them to be removed, there are at least two approaches available to you:
+
+  * You may follow GitHub's [content removal request processes](https://docs.github.com/en/site-policy/content-removal-policies/submitting-content-removal-requests).
+
+  * Alternatively you may contact us (the software developers of this library) and/or file a bugreport with us -- preferably from an email address associated with the recipe website, or otherwise proving ownership of the content you'd like us to remove.  Note that removal of test HTML may reduce our ability to support the affected website(s).
```

Comment:
Actually, hold on.  I think we should restrict this to a subset of developers who actively maintain the library.

Perhaps a clear way to do this would be to suggest contacting any of `maintainers` listed in `pyproject.toml` (I suggest that because I haven't had contact with Hristo recently, and don't want to remove him from the `authors` list, but also want to provide a clear and straightforward place to list contact folks.. `maintainers` seems accurate for that purpose, so I'd add myself there too).

Reply:
Added myself to the `maintainers` list in `pyproject.toml` in 6a6a9144c2be5ce299ce3805ea9344e34afadaca and updated the text here to replace 'software developers' with 'software maintainers'.